tests/e2e/util.go

```diff
@@ -7618,3 +7618,44 @@ func getPortNumAndIP(ip string) (string, string, error) {
 
 	return ip, port, nil
 }
+
+/*
+This function is to scale up and  verify the STS in VKC env
+*/
+func scaleUpAndVerifySts(ctx context.Context,
```

Comment:
We have already available utils for these tasks, see if you can use those -performScalingOnStatefulSetAndVerifyPvNodeAffinity()
createStafeulSetAndVerifyPVAndPodNodeAffinty()
scaleDownStatefulSetPod()
scaleUpStatefulSetPod()